### DISCUSSION TITLE
AddIncludesToQuery does not take place when allowed includes are not provided

### DISCUSSION THREAD

**Discussion by karinarastsinskagia** [spatie/laravel-query-builder#930] [General]
Hello,

I recently discovered that, when I use the `QueryBuilder` for a Model, where I do not want to allow any included fields,  
I do not receive the expected exception error message, if an include input is defined in the query.

For example, suppose that we have defined the following query builder for our model :

`QueryBuilder::for(Article::class)->allowedSorts(['id']);`

In this case, we have not set `allowedIncludes` options. So, if we receive a query like this :

`GET /articles/1?include=comments`

We would expect to get an invalid includes query exception. However, this is not happening since the `AddsIncludesToQuery` trait is not used.

So, I would like to ask , if this is the proper behaviour or if this is something we can add support for? To keep receiving exceptions regardless of not using the `allowedIncludes` method.

Thank you in advance.

PS. The same question is about the `allowedSorts`.

**Comment by KarinaRashchynskaya** [comment]
Hello,

Do you have any feedback for the above?

Thank you
